lib/sales_analyst.rb

```diff
@@ -45,4 +47,11 @@ def average_average_price_per_merchant
     total_averages = BigDecimal(expected, 5).to_s("F")
     total_averages.to_f.floor(2)
   end
+
+  def golden_items
+    expected = @sales_engine.items.item_list.find_all do |item|
+      (item.unit_price_to_dollars / 1000.0) >= above_average
```

Comment:
It seems like the functionality is missing here for what we were trying to calculate? Maybe do another push up to gitHub after you make the commit and save the data